spec/features/root_spec.rb

```diff
@@ -11,6 +11,7 @@
   scenario "displays creation links for administrators" do
     click_link "Add"
     expect(page).to have_link "New Scanned Resource"
+    expect(page).to have_link "New Media Reserve"
```

Comment:
Can you add `href: '/concern/scanned_resources/new/media_reserve` to this?

Reply:
yes I will add it